src/messages/Publication.ts

```diff
@@ -21,22 +21,34 @@ export type PublicationHandler = (publication: Publication) => void;
 
 /**
  * A message used to publish a generic messages
- * to the host application.
+ * from the host application.
  */
 export interface LabeledPublication extends LabeledMsg {
   msgType: 'publish';
   msg: Publication;
 }
 
+/**
+ * A message used to publish a generic messages
+ * to the host application.
+ */
+export interface LabeledClientPublication extends LabeledPublication {
```

Comment:
Can no longer reuse the same interface for host-to-client and client-to-host, unless we wanted to make the client id into sender id.